_dashboards/discover/exploring-query-enhancements.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: default
+title: Exploring query enhancements
+parent: Analyzing data
+nav_order: 20
+---
+
+# Exploring query enhancements
+Introduced 2.18
+{: .label .label-purple }
+
+Starting with OpenSearch 2.17, query enhancements have been made. These enhancements are experimental and may be subject to change or instability. Enhancements include the following: 
+
+- Query languages PPL and SQL, with **Query Assist** for PPL
+- Multiline query editor for PPL and SQL and autocomplete for PPL and DQL
+- Data selector with **Data Explorer** that supports index patterns, indexes, and Amazon S3 connections and data configuration that supports selecting the appropriate query language based on the data type
+- Link sharing through URLs without needing write permission to create an index pattern
+
+All existing functionality remains in OpenSearch 2.17, and the new features are designed to improve your data exploration experience. The **Enable query enhancements** is currently not on a minimal distribution. In a minimal distribution of OpenSearch and OpenSearch Dashboards, query enhancements do not provide access to PPL or SQL functionality. To use these features, the [OpenSearch SQL plugin]({{site.url}}{{site.baseurl}}/search-plugins/sql/settings/) is required. 
+
+Query enhancements work with a basic OpenSearch installation, but to enable SQL, PPL, and external data source queries, additional plugins such as the SQL plugin are necessary. Ensure that you have the [required plugins]({{site.url}}{{site.baseurl}}/install-and-configure/plugins/) installed to take full advantage of these query enhancement features. 
```

Comment:
Perhaps I'm being overly cautious since you did mention it. But did we want to add a sentence that really highlights that it is the external data source that needs the SQL plugin installed. So each one of the data sources need to have the SQL plugin installed. not just the default cluster that they run their usual queries.